bower.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "sb-admin",
+  "version": "0.0.0",
+  "dependencies": {
+    "angular": "1.2.16",
+    "json3": "~3.3.1",
+    "es5-shim": "~3.1.0",
+    "angular-resource": "1.2.16",
+    "angular-cookies": "1.2.16",
+    "angular-sanitize": "1.2.16",
+    "angular-animate": "1.2.16",
+    "angular-touch": "1.2.16",
+    "angular-route": "1.2.16",
+    "font-awesome": "4.3.0",
+    "angular-bootstrap": "0.12.0",
+    "oclazyload": "~0.5.2",
+    "angular-loading-bar": "~0.7.0",
+    "angular-ui-router": "~0.2.13",
+    "angular-toggle-switch": "~1.2.1",
+    "metisMenu": "~1.1.3",
+    "angular-chart.js": "~0.5.2",
+    "d3": "^3.3.13",
+    "metisMenu":"1.1.3",
+    "nvd3": "^1.7.1",
+    "leaflet": "0.7.x",
+    "jquery": "1.9.1 - 2",
+    "angular-nvd3":"^1.0.7",
+    "angular-leaflet-directive": "^0.10.0",
+    "angular-cookies": "1.2.16"
+    
```

Comment:
Hi,
   Is Material as responsive on different screens as bootstrap is?  I have
had prior bootstrap experience so did not give much thought on changing
it.  If it is absolutely necessary then I guess I will have to.  But I have
not done much on material.
On Jun 1, 2016 1:42 PM, "matthesrieke" <notifications@github.com> wrote:

> In bower.json
> <https://github.com/enviroCar/enviroCar-website-ng/pull/2#discussion_r65318605>
> :
>
> > +    "angular-bootstrap": "0.12.0",
> > +    "oclazyload": "~0.5.2",
> > +    "angular-loading-bar": "~0.7.0",
> > +    "angular-ui-router": "~0.2.13",
> > +    "angular-toggle-switch": "~1.2.1",
> > +    "metisMenu": "~1.1.3",
> > +    "angular-chart.js": "~0.5.2",
> > +    "d3": "^3.3.13",
> > +    "metisMenu":"1.1.3",
> > +    "nvd3": "^1.7.1",
> > +    "leaflet": "0.7.x",
> > +    "jquery": "1.9.1 - 2",
> > +    "angular-nvd3":"^1.0.7",
> > +    "angular-leaflet-directive": "^0.10.0",
> > +    "angular-cookies": "1.2.16"
> > +
>
> I am missing angular material here. Is the template based on
> angular-bootstrap? Is it hard to transit to angular material? Let's decide
> whether to got for material or bootstrap. I am a little biased to material
> because I recently did two projects with it.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/enviroCar/enviroCar-website-ng/pull/2/files/d63b4278eaafd713bb9fccc82b8b9af95673f6a2#r65318605>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGszKN0nfvFzCJcUSiHGilfA0xVent4sks5qHT7dgaJpZM4IqxXH>
> .
>